RealestateApi/appsettings.json

```diff
@@ -1,9 +1,13 @@
 {
+  "TypeDatabase": "Postgresql",
+  "ConnectionStrings": {
+    "Postgresql": "Server=localhost;Port=5432;Database=RealEstate;User Id=postgres;Password=root;"
```

Comment:
connection string has to be moved to secrets

Reply:
Yeah because i created this branch from the Create DB 😄